modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="adminnetworkpolicy_{context}"]
+= AdminNetworkPolicy
+
+An `AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definition (CRD). As a {product-title} administrator, you can use ANP to secure your network by creating network policies before creating namespaces. Additionally, you can create network policies on a cluster-scoped level that is non-overridable by `NetworkPolicy` objects.
+
+The key difference between `AdminNetworkPolicy` and `NetworkPolicy` objects are that the former is for administrators and is cluster scoped while the latter is for tenant owners and is namespace scoped.
+
+An ANP allows administrators to specify the following:
+
+* A `priority` value that determines the order of its evaluation. The lower the value the higher the precedence.
+
+* A `subject` that consists of a set of pods that is selected by the policy.
```

Comment:
Subject has namespace so set of pods that are in the selected namespace.
<pre>
subject:
    namespaces:
      matchLabels:
          kubernetes.io/metadata.name: test
</pre>

Reply:
* A `subject` that consists of a set of namespaces or namespace. is what you have for BANP.

Reply:
Should it be namespaces, technically and no namespace (singular)?

Reply:
namespaces: {} refers to all so it can be namespaces or namespace as narrowed down with matchLabels.
Also same label can be applied to more than one namespace. 
Therefore **that consists of a set of namespaces or namespace** is good.